src/App/Pages/Accounts/LoginPasswordlessRequestViewModel.cs

```diff
@@ -306,7 +306,7 @@ private async Task CreatePasswordlessLoginAsync()
             await _deviceActionService.HideLoadingAsync();
         }
 
-        private async Task HandlePasswordlessLogin(PasswordlessLoginResponse response, bool createPendingAdminRequest )
+        private async Task HandlePasswordlessLogin(PasswordlessLoginResponse response, bool createPendingAdminRequest)
```

Comment:
⛏️ Add async suffix:
```suggestion
        private async Task HandlePasswordlessLoginAsync(PasswordlessLoginResponse response, bool createPendingAdminRequest)
```

Reply:
resolved by merge